app/controllers/api/v1/greetings_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    @greetings = @greetings.map(&:greeting)
+    @greetings = @greetings.sample
+    sleep 1
+    render json: { greeting: @greetings }
+  end
+end
```

Comment:
You seem to have the following error please look at the screen shot below:

![Screenshot 2023-05-10 at 7 30 27 PM](https://github.com/gauravgangwarweb/hello-rails-react/assets/79506105/fde5ed93-3406-426f-be81-7e5f683fc782)

Reply:
Hii @KayLemba 
There is no new post controller than how can this error occur